### PR TITLE
added advanced search feature

### PR DESCRIPTION
**Pull request by mahadwaseem123**
Hi @breck7 , current search is now improved with more advance search. Kindly have a review. Refers to issue #121

### CODE REVIEW

**Comment by breck7** [comment]
Thanks @mahadwaseem123 ! This is a good start. I will add more feedback later.

**Comment by mahadwaseem123** [comment]
Hi @breck7 , you'r welcome and sure.

**Comment by breck7** [comment]
No not yet @mahadwaseem123 .

**Comment by breck7** [comment]
I am experimenting with a few options for advanced search. Will share more soon.

File: database/things/mysql.pldb [hunk 1]
```diff
@@ -62,6 +62,7 @@ rijuRepl https://riju.codes/mysql
  example
   SELECT 'Hello, world!';
   
+
```
Comment (breck7) [inline database/things/mysql.pldb:65]: lets see if we can remove these and keep the PRs really focused on the meaningful changes

**Comment by mahadwaseem123** [comment]
> I am experimenting with a few options for advanced search. Will share more soon.

Sure @breck7

File: database/things/mysql.pldb (hunk 1 above)
Comment (mahadwaseem123) [reply database/things/mysql.pldb:65]: @breck7 , these changes are the result of npm run format so I pushed these too.

Comment (breck7) [reply database/things/mysql.pldb:65]: Got it! No problem then.

**Comment by breck7** [comment]
Closing this. I am going to take a different approach for advanced search. Will post more soon.

**Pull request closed without merging**